index.html

```diff
@@ -1236,7 +1307,7 @@ <h4>Update</h4>
                         <h4>Deletion</h4>
                         <p>
                             <span class="rfc2119-assertion" id="tdd-reg-delete">
-                                A TD MUST be removed from the directory when an HTTP `DELETE` request is submitted
+                                A delete operation MUST be done using an HTTP `DELETE` request is submitted
```

Comment:
strike "is submitted" or "MUST be done when an HTTP request is submitted"

Reply:
FIxed